jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -301,6 +301,20 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    // TODO: this should be part of an EndpointMessageTransport.EventHandler interface
+    override fun endpointMessageTransportConnected() {
+        sendVideoConstraints(this.maxReceiverVideoConstraints)
```

Comment:
Is the `this` needed?

Reply:
Nope, changed this to an expression body as well.